src/snappyer.app.src

```diff
@@ -1,7 +1,7 @@
 { application
 , snappyer
 , [ {description,"Snappy as nif for Erlang"}
-  , {vsn,"1.1.3-1.0.5"}
```

Comment:
1.1.3 is the google/snappy version.
please change it to 1.2.0 instead.
(it should be 1.0.6 as a semver bump, but I guess 1.2.0 is less confusing as it is alphabetically  sorted after 1.1.3)